packages/gatsby-source-filesystem/src/create-file-node.js

```diff
@@ -1,4 +1,4 @@
-const slash = require(`slash`)
+const slash = require(`slash2`)
```

Comment:
I think we still require `slash`, even for `slash2`.

Reply:
The author has forked `slash`, added their fix and published as `slash2`